src/task/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -76,6 +76,8 @@
   "loc.messages.metrics.inputs.adjustingAlwaysCloseComment.comment": "The message printed when the always-close-comment mode input is being adjusted to the default.",
   "loc.messages.metrics.inputs.adjustingBaseSize": "Adjusting the base size input to '%s'.",
   "loc.messages.metrics.inputs.adjustingBaseSize.comment": "The message printed when the base size input is being adjusted to the default. The placeholder will be replaced by the default value.",
+  "loc.messages.metrics.inputs.adjustingChangeTitle": "Adjusting the change-title mode input to 'false'.",
```

Comment:
Looking at the code, I see that `adjustingChangeTitle` is used for the `true` case while `disablingChangeTitle` is used for the `false` case, so that the messages are the wrong way around.

I'd favour keeping the ordering in the code and making the correction in this file as disabling corresponds in my mind to `false`.

Please update the comments to reflect this change too.